packages/web/src/pages/pay-and-earn-page/mobile/PayAndEarnPage.tsx

```diff
@@ -138,7 +138,13 @@ export const PayAndEarnPage = ({ tableView }: PayAndEarnPageProps) => {
         <LoadingSpinner className={styles.spinner} />
       ) : (
         <>
-          <USDCCard balance={balance} />
+          <USDCCard
+            balance={balance}
+            refreshing={
+              balanceStatus === Status.LOADING ||
```

Comment:
ditto `StatusIsNotFinalized`?

Reply:
Same response as the other instance: It's an uncommon case where I think we only want this to be true in `LOADING` states.

Reply:
I'll also add that there will be follow up work to change how we show the USDC balance based on loading/refreshing states. So I'll be revisiting this soon!